CONTRIBUTING.md

```diff
@@ -31,7 +31,28 @@ A maintainer will request changes if necessary. For content contributions, don't
 
 ## Setup Instructions
 
-This project is built using [Hugo](https://gohugo.io/), an open-source static site generator. To get the project up and running:
+This project is built using 
+- [Hugo](https://gohugo.io/), an open-source static site generator.
+- [Docker](https://www.docker.com/), a platform to generate containerized environments.
+
+Using Docker for the project is preferred to minimize conflicts arising from machine, OS, or dependency variations.
+
+### To get the project up and running Docker:
+1. Download and Install [Docker](https://docs.docker.com/get-docker/)
+2. Run `docker compose up`
+3. Visit the site at http://localhost:1313
+4. Stop the container with `Ctrl + C` or `docker compose down`
+
+    **NOTE:** This repo uses [Google's reCaptcha](https://developers.google.com/recaptcha/docs/faq#im-getting-an-error-localhost-is-not-in-the-list-of-supported-domains.-what-should-i-do). To view the recaptcha in development you can request your own key, list `127.0.0.1` as the domain and visit the site on the browser at `127.0.0.1:1313`. The datasite key can be edited at `themes/hugo-theme-ephemeralenvironments/layouts/partials/subscribe.html`
```

Comment:
I have since added `localhost` to the list of supported domains, but this is very helpful! Maybe I would pivot this to say "If you're using your own sitekey, add `localhost` to the list of supported domains to test your Google reCAPTCHA locally"

Reply:
Can you actually test this for me? ie try to access the site via localhost, and have the reCaptch supported domains only reference localhost. I am pretty sure that you need to hardcode the IP address as 127.0.0.1. I initially tried localhost, but got an error, will have to see if I can regenerate the error on my end.